aspnetcore/tutorials/min-web-api.md

```diff
@@ -875,15 +875,63 @@ The following image shows the Postman update:
 
 The sample app implements a single DELETE endpoint using `MapDelete`:
 
-[!code-csharp[](min-web-api/samples/6.x/todo/Program.cs?name=snippet_delete)]
+[!code-csharp[](min-web-api/samples/7.x/todo/Program.cs?name=snippet_delete)]
 
 Use Postman to delete a to-do item:
 
 * Set the method to `DELETE`.
 * Set the URI of the object to delete (for example `https://localhost:5001/todoitems/1`).
 * Select **Send**.
 
-<a name="over-post-v6"></a>
+## Use the MapGroup API
+
+The sample app code repeats the `todoitems` URL node each time it sets up an endpoint. Web APIs often have groups of endpoints with a common URL node, and the <xref:Microsoft.AspNetCore.Builder.EndpointRouteBuilderExtensions.MapGroup%2A> method is available to help organize such groups. It reduces repetitive code and allows for customizing entire groups of endpoints with a single call to methods like <xref:Microsoft.AspNetCore.Builder.AuthorizationEndpointConventionBuilderExtensions.RequireAuthorization%2A> and <xref:Microsoft.AspNetCore.Builder.RoutingEndpointConventionBuilderExtensions.WithMetadata%2A>.
+
+Replace the code in Program.cs with the following code:
+
+:::code language="csharp" source="min-web-api/samples/7.x/todoGroup/Program.cs" id="snippet_all":::
+
+Notice that a `MapGroup()` call sets up the group, and the endpoints omit the `todoitems` prefix:
```

Comment:
Edit: I like `Map<HttpVerb>`

Consider changing this to something like

The preceding code has the following changes:

* Add `RouteGroupBuilder group = app.MapGroup("/todoitems");` to set up the map group using the URI prefix `/todoitems`
* Changes all the `app.MapVerb` methods to :
   *  `group.MapVert`
   * Removes the mapped URI prefix `/todoitems` from the `MapVerb` methods first parameter.
 
To keep it that short and clean, you'd probably have to remove `app.MapGet("/", () => "Hello World!");` so you don't have to explain that away. But that would be an improvement. Otherwise you're just showing a bunch of code and it's not clear what has changed.

Reply:
If we get rid of the Hello world MapGet we lose the “browser check” feature of the tutorial sample app and when we run the app we’ll get an error page instead of a Hello world message.

Reply:
Let's see how it goes. API code doesn't have a UI. Makes for a cleaner migration if you ask me.

Reply:
Yes, it looks good without extra MapGet.